Add guidance for standard / custom catalogs (#901)
chainguard-dev/edu: content/chainguard/chainguard-images/using-the-tag-history-api.md

And your image will then be locked into that specific build of the `python:latest` image variant.

## Accessing the History API for Standard and Custom Images

> If you are a customer who would like to access the history API for the Standard Catalog or a Custom Catalog, you can create credentials to store the credentials in order to access the API with curl and Bash. 

Reviewer: ```suggestion
If you are a customer who would like to access the history API for the Standard Catalog or a Custom Catalog, you must create credentials to access the API with curl and Bash. 
```

I think?

### Create Credentials

You'll need to use credentials to access the Registry. First, authenticate with `chainctl`.

```sh
chainctl auth login
```

Next, grab your Group ID and store it in a variable.

```sh
chainctl iam groups ls -o table
export GROUP_ID="cc...52"
```

At this point, you can create a token for 30 days. Note that 30 days is the default, the max you can initialize a token for is 1 year.

```sh
chainctl auth configure-docker --pull-token --group $GROUP_ID --ttl 720h
```

If you would like to use this pull token in another environment, run this command with Docker.

```sh
docker login "cgr.dev" --username "cc..96" --password "eyJ...X34"
```

Be sure to keep a copy of the username and password created.

### Optional: Storing Credentials 

> The username and password can be used if required, but if you’d like to use the default Docker functionality, then this is base64 encoded and stored within `~/.docker/config.json`. 

Reviewer: ```suggestion
You can use the username and password you just created if required. However, Docker's default functionality is to encode these credentials in base64 and then store them within `~/.docker/config.json`. 
```

I thought this was a little clearer
